server/src/main/kotlin/io/spine/server/entity/TransactionalEntityExts.kt

```diff
@@ -95,3 +96,45 @@ public fun <I, E : TransactionalEntity<I, S, B>, S : EntityState<I>, B : Validat
     val builder = builder()
     block(builder)
 }
+
+/**
+ * Obtains the entity identifier.
+ *
+ * This is a shortcut for `id()`.
+ *
+ * @param I the type of the entity identifiers.
+ * @param E the type of the transactional entity.
+ * @param S the type of the entity state.
+ * @param B the type of the entity state builder.
+ */
+public val <I, E : TransactionalEntity<I, S, B>, S : EntityState<I>, B : ValidatingBuilder<S>>
+        E.id: I
+    get() = id()
+
+/**
+ * Obtains the entity version.
+ *
+ * This is a shortcut for `version()`.
+ *
+ * @param I the type of the entity identifiers.
+ * @param E the type of the transactional entity.
+ * @param S the type of the entity state.
+ * @param B the type of the entity state builder.
+ */
+public val <I, E : TransactionalEntity<I, S, B>, S : EntityState<I>, B : ValidatingBuilder<S>>
+        E.version: Version
+    get() = version()
+
+/**
+ * Obtains the entity state.
+ *
+ * This is a shortcut for `state()`.
+ *
+ * @param I the type of the entity identifiers.
+ * @param E the type of the transactional entity.
+ * @param S the type of the entity state.
+ * @param B the type of the entity state builder.
+ */
+public val <I, E : TransactionalEntity<I, S, B>, S : EntityState<I>, B : ValidatingBuilder<S>>
+        E.state: S
+    get() = state()
```

Comment:
This does not look like a property to me. The access to the value is synchronized in `AbstractEntity`.

Reply:
Yes, but we want it for Kotlin code, in tests in particular.